pubspec.yaml

```diff
@@ -77,7 +77,10 @@ dependencies:
   punycode: ^1.0.0
   qr_code_scanner: ^1.0.0
   qr_flutter: ^4.0.0
-  receive_sharing_intent: ^1.4.5
+  receive_sharing_intent:
+    git:
+      url: git@github.com:krabbenprgr/receive_sharing_intent.git
+      ref: receive-.txt-v2
```

Comment:
why we need this?

Reply:
The package is not maintained by the author. In the repository, it has a lot of PRs pending and nice features, so this PRs allow share .txt files.